src/content/services/monetizationTagManager.ts

```diff
@@ -216,8 +216,8 @@ export class MonetizationTagManager extends EventEmitter {
       }
     }
 
-    this.sendStartMonetization(startMonetizationTags)
     this.sendStopMonetization(stopMonetizationTags)
+    this.sendStartMonetization(startMonetizationTags)
```

Comment:
Question: I think we should `await` the stop and then run the start. I'm worried about race conditions.